src/components/Header/Branding.js

```diff
@@ -58,9 +58,8 @@ const TopNavBranding = ({ slug }) => (
       });
 
       let {
-        type,
         fillColorInitial = logoStyleDefaults.fillColorInitial,
-        fillColorHover = fillColorInitial,
+        fillColorHover,
```

Comment:
Setting defaults again here is so that you can do partial overrides using a context provider. If we do `fillColorHover = logoStyleDefaults.fillColorHover` here now, we can omit the `fillColorHover: colors.grey,` in the context providers, and it won't break if you introduce a context provider without a `fillColorHover` in the future :)
(This is because React does not merge object properties on context value like it does for state values)